cmd/openshift-install/destroy.go

```diff
@@ -58,11 +62,13 @@ func newDestroyClusterCmd() *cobra.Command {
 			logrus.Infof("Uninstallation complete!")
 		},
 	}
+	destroyCmd.Flags().BoolVar(&deleteVolumes, "delete-volumes", false, "Delete CNS volumes on destroy.")
```

Comment:
@patrickdillon installer doesn't have flags usually, any recommendation for alternative here?